src/main/java/ro/unibuc/triplea/infrastructure/SecurityConfiguration.java

```diff
@@ -26,7 +26,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         http
                 .csrf(csrf -> csrf.disable())
                 .authorizeHttpRequests(auth -> auth
-                        .requestMatchers("/api/v1/auth/**", "/v3/api-docs/**", "/swagger-ui/**").permitAll()
+                        .requestMatchers("/api/v1/auth/**", "/v3/api-docs/**", "/swagger-ui/**", "/api/v1/games/**").permitAll()
```

Comment:
/api/v1/games/** have access everyone. do we want that? i think an authenticated user (eventually based on role) should have access to those endpoints.

Reply:
Here I think we can allow everyone, since those endpoints are only for listing games. If we had a FE I'd imagine we could have this info on a main page available for everyone, including unauthenticated users. The next endpoints we make though, we should leave behind security (reviews/favourites)

Reply:
It remains public as the team agreed.